src/main/kotlin/no/nav/klage/oppgave/service/OppgaveService.kt

```diff
@@ -86,6 +88,35 @@ class OppgaveService(
         )
     }
 
+    fun searchOppgaverByFagsakId(fagsakId: String): SearchSaksnummerResponse {
+        val data = behandlingRepository.findByFagsakId(fagsakId = fagsakId)
+        val paaVentBehandlinger = mutableListOf<UUID>()
+        val feilregistrerteBehandlinger = mutableListOf<UUID>()
+        val avsluttedeBehandlinger = mutableListOf<UUID>()
+        val aapneBehandlinger = mutableListOf<UUID>()
+
+        data.forEach {
+            if(!tilgangService.harInnloggetSaksbehandlerTilgangTil(it.sakenGjelder.partId.value)) {
```

Comment:
Kan saker med samme `fagsakId` gjelde ulike `sakenGjelder`? Hvis ikke, kan dette forbedres ytelsemessing.

Reply:
Iom. at `fagsakId` bare er unikt innenfor hvert enkelt fagsystem, så tenker jeg at det er mulighet for ulike `sakenGjelder`. Det er også cache på eksterne kall som følger denne.

Reply:
Hvis vi tror det kan være ulike `sakenGjelder`, så tror jeg vi burde unngå å sjekke samme `fnr` flere ggr grunnet prisen.